### PR TITLE
Replace telescope icon to ember logo

### PR DESCRIPTION
**Pull request by kaermorchen**
ember-icon

### CODE REVIEW

**Comment by lifeart** [comment]
@kaermorchen I think we could remove "Ember" text for "loaded" state

**Comment by kaermorchen** [comment]
@lifeart Done.

**Comment by lifeart** [comment]
@kaermorchen I mean "loaded" state where we have icon + Ember text,  not loading state, loading state with Ember text is completely fine

File: src/extension.ts [hunk 1]
```diff
@@ -128,9 +128,9 @@ export async function activate(context: ExtensionContext) {
   // client can be deactivated on extension deactivation
   context.subscriptions.push(
     commands.registerCommand(ELS_COMMANDS.SET_STATUS_BAR_TEXT, async () => {
-      ExtStatusBarItem.text = "$(telescope) " + 'Reloading projects...';
+      ExtStatusBarItem.text = "$(ember-logo) " + 'Reloading projects...';
       await commands.executeCommand(ELS_COMMANDS.RELOAD_PROJECT);
-      ExtStatusBarItem.text = "$(telescope) " + 'Ember';
+      ExtStatusBarItem.text = "$(ember-logo) " + 'Ember';
```
Comment (lifeart) [inline src/extension.ts]: here we could remove "Ember"

File: src/extension.ts [hunk 2]
```diff
@@ -226,7 +226,7 @@ export async function activate(context: ExtensionContext) {
         return commands.executeCommand(command, ...args);
       });
       commands.executeCommand(ELS_COMMANDS.SET_CONFIG, config);
-      ExtStatusBarItem.text = "$(telescope) " + 'Ember';
+      ExtStatusBarItem.text = "$(ember-logo) " + 'Ember';
```
Comment (lifeart) [inline src/extension.ts]: here we could remove "Ember"

File: src/worker-extension.ts [hunk 3]
```diff
@@ -95,10 +95,10 @@ export async function activate(context: ExtensionContext) {
   // client can be deactivated on extension deactivation
   context.subscriptions.push(
     commands.registerCommand(ELS_COMMANDS.SET_STATUS_BAR_TEXT, async () => {
-      ExtStatusBarItem.text = "$(telescope) " + 'Reloading projects...';
+      ExtStatusBarItem.text = "$(ember-logo) " + 'Reloading projects...';
       const result = await commands.executeCommand(ELS_COMMANDS.RELOAD_PROJECT, workspace.rootPath, true);
       window.showInformationMessage(JSON.stringify(result));
-      ExtStatusBarItem.text = "$(telescope) " + 'Ember';
+      ExtStatusBarItem.text = "$(ember-logo) " + 'Ember';
```
Comment (lifeart) [inline src/worker-extension.ts]: here we could remove "Ember"

File: src/worker-extension.ts [hunk 4]
```diff
@@ -138,7 +138,7 @@ export async function activate(context: ExtensionContext) {
       collectTemplateTokens: true,
       useBuiltinLinting: false
     }});
-    ExtStatusBarItem.text = "$(telescope) " + 'Ember';
+    ExtStatusBarItem.text = "$(ember-logo) " + 'Ember';
```
Comment (lifeart) [inline src/worker-extension.ts]: here we could remove "Ember"

**Comment by kaermorchen** [comment]
@lifeart like this?

Безымянный

**Review by lifeart — APPROVED** [review]

**Comment by lifeart** [comment]
@kaermorchen yep, all good, thank you!

**Pull request merged**